src/content/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/default-metrics-and-metric-attributes.mdx

```diff
@@ -0,0 +1,326 @@
+---
+title: 'Default metric labels and metric attributes'
+tags:
+  - Integrations
+  - Prometheus integrations
+  - Install and configure Prometheus agent
+metaDescription: 'TODO'
+---
+
+# Metric labels [#metric-labels]
+
+Labels are metadata or attributes, and there are a few sources of metric labels. In the following tables, see the sources of labels and when they are added:
+
+<CollapserGroup>
```

Comment:
the table is not easy to be read due to the size of the columns, could we do something?

<img width="411" alt="Screenshot 2022-11-11 at 14 45 33" src="https://user-images.githubusercontent.com/43335750/201353009-bda9390b-a2dc-44bf-86f0-f245027323cd.png">